From 0e62490e0bf14cbe24df85f1c173bdbd617e48f0 Mon Sep 17 00:00:00 2001
From: Hong-Mei Li <a21834@motorola.com>
Date: Wed, 23 Sep 2015 23:18:18 -0700
Subject: [PATCH 178/959] IKSWM-4458 moto:defconfig: trigger panic when RT
 throttle

Enable the kernel to panic when a realtime runqueue is throttled.

Change-Id: Ie992d6dd586faee7df152b8d40b24b96eea3db4a
Signed-off-by: Hong-Mei Li <a21834@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index fc88241834b..6df2301857b 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -25,3 +25,4 @@ CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_FW_UPDATE_MMI=y
 CONFIG_MOT_UTAG=y
 CONFIG_PROC_DEVICETREE=y
 CONFIG_DIAG_OVER_TTY=y
+CONFIG_PANIC_ON_RT_THROTTLING=y
-- 
2.11.0

